home-application/src/main/resources/config/application.properties

```diff
@@ -1,3 +1,4 @@
+#spring.profiles.active=dev
```

Comment:
We are going to delete dataloader and we don't need this profile

Reply:
Agree But now dataloader is in the project becouse I did not delete this commented code